kivy/input/providers/hidinput.py

```diff
@@ -548,22 +568,40 @@ def process_as_mouse_or_keyboard(
                                 point['id'] += 1
                                 point['_avoid'] = True
                     else:
+                        if ev_value < 0 or ev_value > 1:
```

Comment:
i would have rather written it this way
```suggestion
                        if not 0  <= ev_value <= 1:
```
but seems that produces more opcodes (26 instead of 21), so your version might be faster, not a biggie anyway.